include/fiction/algorithms/path_finding/distance.hpp

```diff
@@ -62,6 +64,30 @@ template <typename Lyt, typename Dist = double>
     return static_cast<Dist>(std::hypot(x, y));
 }
 
+/**
+ * The Euclidean distance between two SiDBs on the H-Si surface (SiQAD coordinates are required).
+ * In the first step, SiQAD coordinates are converted to a location on the Si-substrate by taking Silicon's lattice constants into account (lat_a, lat_b, lat_c. Afterward, the Euclidean distance is calculated.
```

Comment:
`lat_a`, `lat_b`, `lat_c` do not appear in the code. What does that mean? Also, missing `)` in the text

Reply:
You are right, but they occur in the "real_position" function and are definded in the simulation_parameter struct

Reply:
Then please reference it here